### ISSUE TITLE
Robustness Benchmarking

### ISSUE THREAD

**Issue by ebridge2** [neurodata/lol#7]
compare robust PCA and LDA on benchmarks

add results to figure 5

add a new panel for a scatter plot

add table with percent of points in each quadrant

add annotations to the scatterplot

cross: pick 2 dimensions at change point, trunk-3 pick first dimension, robust pick 2 points somewhere in the middle

**Comment by jovo** [comment]
yes, though i wouldn't try to add a new panel to fig 5 yet, let's just look at it, and then we can decide what to do with it.

**Comment by ebridge2** [comment]
table names; figure -> LOL better/worse in r/L
table corresponds to the marginals themselves
table with panel A; B and C below
table doesn't have own letter
error instead of misclassification?
table is marginals + LOL dominating
